azure-pipelines.yml

```diff
@@ -0,0 +1,150 @@
+trigger:
+- release/*
+  
+pool:
+  vmImage: windows-latest
+
+parameters:
+- name: doPublish
+  default: false
+  type: boolean
+- name: toDeploy
+  default: false
+  type: boolean
+
+variables:
+- name: buildConfiguration
+  value: 'Release'
+
+resources:
+  repositories:
+  - repository: inss-devops-common-lib
+    type: github
+    endpoint: InsolvencyService
+    name: InsolvencyService/inss-devops-common-lib
+    ref: develop
+    
+stages:
+- stage: BuildTest
+  displayName: 'Build and Test'
+  jobs:  
+    - job: 
+      steps:
+      - task: UseDotNet@2
+        inputs:
+          version: '6.0.x'
+
+      - task: DotNetCoreCLI@2
+        displayName: NuGet Restore
+        inputs:
+          command: 'restore'
+          projects: '**/INSS.FIP.sln'
+  
+      - task: DotNetCoreCLI@2
+        displayName: 'Build Solution'
+        inputs:
+          command: 'build'
+          arguments: '--configuration $(buildConfiguration)'        
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Run Tests'
+        inputs:
+          command: test
+          projects: '**/*Tests/*.csproj'
+          arguments: '--configuration $(buildConfiguration) --collect "Code coverage"'
+
+      - task: DotNetCoreCLI@2
+        displayName: 'Publish Project'
+        inputs:
+          command: 'publish'
+          publishWebProjects: false
+          projects: '**/*.csproj'
+          arguments: '--configuration $(BuildConfiguration) --output $(Build.ArtifactStagingDirectory)'
+        condition: eq('${{ parameters.doPublish}}', 'true')
```

Comment:
No need to have a condition on this - we'd always want to make sure the publish step works, even if we don't save the resulting artefact.

Reply:
Removed in latest commit